BUILD

```diff
@@ -70,6 +70,10 @@ chmod 755 dist/envoy.aar
 chmod 755 dist/envoy-pom.xml
 chmod 755 dist/envoy-javadoc.jar
 chmod 755 dist/envoy-sources.jar
+sha256sum dist/envoy.aar | cut -d " " -f1 > dist/envoy.aar.sha256
```

Comment:
Maybe do it in Python script instead of doing `cut -d " "` stuff

Reply:
I can actually just have the python script generate the hashes

Reply:
That's what I meant :)